### ISSUE TITLE
Moving large items to cargo hold won't let you move the right number of units

### ISSUE THREAD

**Issue by jdhirst** [EvEmu-Project/evemu_Crucible#69]
From AlasiyaEvE created by [ryllae](https://github.com/ryllae): zhyrohaad/AlasiyaEvE#37  

I have 2,033,809 m^3 units of space in my Charon's cargo hold, and a few hundred packaged Rokh's (50Km^3) in my inventory.  If you drag the stack to the charon's cargo hold, it tells you it can only move up to 40.  Click ok, and nothing moves.  By trial and error, I can move up to a stack of *4*, but I get an error that there is insufficient space in the source (?) location if I try to move 5 or more at once:
![Capture33](https://user-images.githubusercontent.com/51723245/63744829-22592e00-c866-11e9-8fab-139592c1daf9.PNG)

**Comment by jdhirst** [comment]
I have a suggestion for how to make this work:  when moving items, first "pick up" the items into a fictitious container (say, an invisible unlimited container hidden in the Session or Client object).  Then attempt to drop the items back into the new container (which will prevent many of the current errors, including #57?).    If it fails to go into the target, revert with no checks on limitations?

You could also check if the "temp container" has something at login and dump it out to a jetcan or station in the event of catastrophic failure during xfer?

**Comment by jdhirst** [comment]
This I haven't noticed before and will have to look at when I get time.

**Comment by jdhirst** [comment]
cargo capy checks and moving items updated

**Comment by jdhirst** [comment]
this seems to be fixed

main is not updated as of yet.

**Comment by jdhirst** [comment]
updating capy checks and item movement between containers
this update will also address bugs #57, #58, and #68.

**Comment by jdhirst** [comment]
Not sure if related to moving large numbers of stuff in jetcans: https://github.com/zhyrohaad/AlasiyaEvE/issues/28

**Comment by jdhirst** [comment]
it was working when i tested on dev server.
will look over it again.

**Comment by jdhirst** [comment]
Still a bug.  You can also create "phantom items", that when moved, crash the server.

**Comment by jdhirst** [comment]
Also of interest: while testing #58, I wasn't able to consistently abuse sizes if the jetcan or cargo hold was exactly full.  Dragging to an empty cargo hold splits the stack twice, apparently trying to move it twice?  You can get some interesting bogus errors depending on the amount of space available in the *source* container too.  I sometimes get the 2nd stack moved instead of the first, which sometimes has the the *sources*'s max size, instead of the destination?  I'll let you take a pass at the damage here before I write up more bugs on this.

**Comment by jdhirst** [comment]
i have rewrote inventory testing code and added many missing checks.
main is updated with this revision.

**Comment by jdhirst** [comment]
yeah, the `split` function uses the 'current' cargohold as a container for items.
this also will do some weird shit with inventory, and ties into the current list of inventory bugs.
will be part of the current inventory test/check/rewrite shit.

**Comment by jdhirst** [comment]
Dragging a huge stack of packaged Rokh's to my charon correctly splits the stack in the expected number (40), but doesn't deposit them in the cargo hold.  I can't move a stack of 40 either.

**Comment by zhyrohaad** [comment]
last two comments are latest.
needs testing to verify status
